content/en/platform/corda/5.0/application-networks/managing/member-suspension/_index.md

```diff
@@ -0,0 +1,94 @@
+* `AUTH_INFO` — the authetication information if using PowerShell. You can set this as follows:
+   ```
+   $AUTH_INFO = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("admin:admin" -f $username,$password)))
+   ```
+
+## Query for Members
+
+You can use the [Member Lookup REST endpoint](../../reference/rest-api/C5_OpenAPI.html#tag/Member-Lookup-API) to query for all members with a particular status by specifying the MGM and the status. By default, the endpoint returns only ACTIVE members. You can also query for multiple statuses together. For example, to query for all members with the status SUSPENDED:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```shell
+curl -u $REST_API_USER:$REST_API_PASSWORD "$REST_API_URL/members/$MGM_HOLDING_ID?statuses=SUSPENDED"
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)}`
+   -Method Get -Uri $API_URL/members/$MGM_HOLDING_ID`?statuses=SUSPENDED
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+The GET method of the `/api/v1/members/{holdingidentityshorthash}` endpoint returns a list of members, specifying the `memberContext` and `mgmContext` of each. You can extract the serial number of a member from the `corda.serial` field inside the `mgmContext`. This serial number should be used when [suspending]({{< relref "#suspending-a-member">}}) or [activating]({{< relref "#activating-a-member">}}) a member.
+
+## Suspend a Member
+
+You can use the <a href="../../reference/rest-api/C5_OpenAPI.html#tag/MGM-API/operation/post_mgm__holdingidentityshorthash__suspend">POST method of the `/api/v1/members/{holdingidentityshorthash}/suspend` endpoint</a> to suspend a member of a group:
+
+{{< tabs >}}
+{{% tab name="Bash"%}}
+```shell
+curl -u $REST_API_USER:$REST_API_PASSWORD -X 'POST' "$API_URL/mgm/$MGM_HOLDING_ID/suspend" -H 'Content-Type: application/json' \
+ -d '{"x500Name": '\"$MEMBER_X500_NAME\"', "serialNumber": "<serial-number>"}'
+```
+{{% /tab %}}
+{{% tab name="PowerShell" %}}
+```shell
+Invoke-RestMethod -Headers @{Authorization=("Basic {0}" -f $AUTH_INFO)}`
+  -Method Post -Uri $API_URL/mgm/$MGM_HOLDING_ID/suspend -Body (ConvertTo-Json -Depth 1 @{
+  x500Name = $MEMBER_X500_NAME; serialNumber = <serial-number>})
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+`<serial-number>` is the current serial number of the member, as retrived by the [Member Lookup endpoint]({{< relref "#query-for-members">}}). The `<serial-number>` is optional. If it is not specified, the latest serial number is used. However, we recommend always specifying the serial number in the request to avoid suspending a member based on outdated information. If the serial number does not match, the method returns a `409 CONFLICT`. This can happen if another process has updated the member information, before the suspension operation. If this occurs, you can query the Member Lookup endpoint again and decide whether to proceed with the operation.
```

Comment:
```suggestion
`<serial-number>` is the current serial number of the member, as retrieved by the [Member Lookup endpoint]({{< relref "#query-for-members">}}). The `<serial-number>` is optional. If it is not specified, the latest serial number is used. However, we recommend always specifying the serial number in the request to avoid suspending a member based on outdated information. If the serial number does not match, the method returns a `409 CONFLICT`. This can happen if another process has updated the member information before the suspension operation. If this occurs, you can query the Member Lookup endpoint again and decide whether to proceed with the operation.
```